src/test/resources/org/rust/ide/formatter/fixtures/macro_use_after.rs

```diff
@@ -0,0 +1,4 @@
+#[macro_use] extern crate foo;
```

Comment:
So we always place `macro_use` inline? I think this is a bit too opinionated: some do this, some don't:

https://github.com/search?utf8=%E2%9C%93&q=%23%5Bmacro_use%5D&type=Code&ref=searchresults

It's better to either have an option here, or to be able to preserve existing formatting.